cfme/containers/provider/__init__.py

```diff
@@ -230,7 +230,11 @@ def num_project(self):
     @num_project.variant('ui')
     def num_project_ui(self):
         view = navigate_to(self, "Details")
-        return int(view.entities.summary("Relationships").get_text_of("Projects"))
+        if self.appliance.version < "5.9":
```

Comment:
Please use the Widgetastic VersionPick class here:
https://github.com/jawatts/widgetastic.core/blob/master/src/widgetastic/utils.py#L241

Example:
```
VersionPick({
        Version.lowest(): 'Projects',
        '5.9': 'Container Projects'
    })
```